package.json

```diff
@@ -56,11 +57,18 @@
         "ngx-mat-select-search": "~4.0.2",
         "ol": "~6.13.0",
         "proj4": "~2.7.5",
+        "propagating-hammerjs": "^2.0.1",
         "rxjs": "~7.5.2",
         "tslib": "^2.3.1",
         "vega": "~5.21.0",
         "vega-embed": "~6.20.5",
         "vega-lite": "~5.2.0",
+        "vis-timeline": "7.4.4",
```

Comment:
whydo we need all the other dependencies? Are they dev dependencies?

Reply:
unfortunately all of them are needed, but some are dev dependencies
5fb3e1db202518d150f0b15d43447b7c83f4c568

Reply:
How can they be dev-dependencies?
For me it seems odd, since they should be used during runtime:
cf. https://github.com/visjs/vis-timeline/blob/master/package.json#L60-L69

Reply:
I guess you are right. I put them back to the dependencies ec74d8089fd6fb32fa6e222c768dcff2770248f1